src/pages/explorer/index.tsx

```diff
@@ -33,4 +40,14 @@ export default function Explorer() {
       </Box>
     </Layout>
   );
-}
\ No newline at end of file
+}
+
+export async function getServerSideProps() {
+  const envs = getClientSideEnvs();
```

Comment:
suggestion: instead of calling this function on every page, expose selected environmental variables with `NEXT_PUBLIC_` prefix. See [nextjs docs about runtime envs](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#runtime-environment-variables).

Reply:
NEXT_PUBLIC_ envs will be compiled within the build unfortunately. So you can not provide an env var starting with NEXT_PUBLIC_  after the build is complete and let nextjs use that.